src/DependencyManagement/DependencyManager.cs

```diff
@@ -42,6 +42,9 @@ internal class DependencyManager
         private const string HomeDriveName = "HOME";
         private const string DataFolderName = "data";
 
+        // AzureFunctions folder name.
+        private const string AzureFunctionsFolderName = "AzureFunctions";
```

Comment:
nit: move this one down next to `ManagedDependenciesFolderName`

Reply:
Fixed.